cpp/CMakeLists.txt

```diff
@@ -46,7 +46,7 @@ set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
 option(BUILD_SHARED_LIBS "Build raft shared libraries" ON)
 option(BUILD_TESTS "Build raft unit-tests" ON)
-option(BUILD_BENCH "Build raft C++ benchmark tests" OFF)
+option(BUILD_BENCH "Build raft C++ benchmark tests" ON)
```

Comment:
Just a note that we probably want to turn this back off to alleviate surprises downstream